src/Bridge/Doctrine/Orm/Cache/QueryExpander.php

```diff
@@ -0,0 +1,112 @@
+<?php
```

Comment:
This class should be an extension that alters the query probably implementing `QueryResult{Collection|Item}ExtensionInterface`.